pkg/controller/managedresource/reconciler.go

```diff
@@ -238,6 +238,12 @@ func (r *Reconciler) reconcile(ctx context.Context, mr *resourcesv1alpha1.Manage
 				newObj.oldInformation, _ = existingResourcesIndex.Lookup(objectReference)
 				decodedObj = nil
 
+				if ignoreMode(obj) {
```

Comment:
I guess, after setting ignore mode, the warning annotations ("do not edit, reconciled by grm...") and origin annotation are still present. This will be a bit confusing, do you agree?

Reply:
Yes, I agree. But I don't find and good way to achieve this for now. It is tricky as GRM needs to remove these annotations only once and it should not remove these annotations when another GRM already manages the corresponding resource.

Reply:
Yeah. grm could take a look at the resources in the status and build up a slice of resources that need to be "orphaned". Meaning, it should loop over all of the resources that specify ignore but are still tracked in the status.
If the origin annotation is still set to its own identity, then it should remove the origin annotation and the description annotation. If the origin annotation has changed, we know that some other grm has already taken over control.
In both cases, it can remove the resource from the status afterwards.

With this, we would have a clear contract / mechanism, how adding the ignore mode works.
WDYT?

Reply:
Yes, makes sense. I now added the corresponding handling. Thank you for your implementation proposal, appreciated it.